Form/Fields/AbstractFormField.cs

```diff
@@ -0,0 +1,81 @@
+﻿using System;
+using DUCK.Form.Validation;
+using DUCK.Utils;
+using UnityEngine;
+
+namespace DUCK.Form.Fields
+{
+	[DisallowMultipleComponent]
+	public abstract class AbstractFormField : MonoBehaviour
```

Comment:
This class has been used quite a lot. Comments?

Reply:
It has comments? on every function except awake